junit5/src/main/java/com/linecorp/armeria/testing/junit5/client/WebTestClientUtil.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.testing.junit5.client;
+
+import static java.util.Objects.requireNonNull;
+
+import com.linecorp.armeria.common.QueryParams;
+import com.linecorp.armeria.common.annotation.Nullable;
+
+final class WebTestClientUtil {
```

Comment:
It seems to be copied from the main source set. You can move the package-private code to the internal package if it is reasonable.

Reply:
Sorry, I'm afraid I didn't quite understand where "the internal package" is.

I copied the WebClientUtil class because it was package-private. Are you suggesting that I move the addQueryParams method into the WebTestClient interface?